components/salsa-macro-rules/src/setup_tracked_fn.rs

```diff
@@ -288,5 +283,10 @@ macro_rules! setup_tracked_fn {
                 }
             })
         }
+
+        #[allow(non_camel_case_types)]
```

Comment:
I would consider adding a comment explaining the importance of why this has to come last to avoid accidentally breaking this in the future.

Reply:
Added a description; I'll land this change as Lukas is on vacation for the next 3 weeks.